app/services/query-service.js

```diff
@@ -113,9 +185,13 @@ const fetchAnswer = async (req, query, chatHistory) => {
     input: query
   })
 
+  // run the validation, don't throw an error if it fails
+  validateResponseLinks(response)
```

Comment:
We're not doing anything with the returned error string, is this intentional?

Reply:
That's actually something I wanted to ask you. As is, the code just triggers the function and logs the problematic response, without filtering it out. Should it? Should it add a frontend warning also (you mentioned that no but I am happy to consult Emma if you prefer)?
@MohammedFoysal

Reply:
lets leave it for now. we have the logged event to find out how often this happens. we can prioritise and address this in a follow up ticket